.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea/
 *.iml
 node_modules/
-.vscode/
\ No newline at end of file
+.vscode/
+.vscode
```

Comment:
I think this change is unnecessary. Should work the way it was too

Reply:
Yep, seems to work. I think someone pushed before was in the .gitignore